src/app/core/header/header.component.test.ts

```diff
@@ -15,4 +17,19 @@ describe('Header', () => {
     it('should show navigation routes', () => {
         expect(sut.navigationRoutes).toEqual(NAVIGATION_ROUTES);
     });
+
+    describe('is option hidden', () => {
```

Comment:
Agree. Up to you. It is just more effective to find corresponding test by method name if needed.